shell/deploy-multi-chain.sh

```diff
@@ -104,15 +103,17 @@ declare -A chains
 # define function to initialize all configurations
 function initialize {
     chains["arbitrum"]="$ARBITRUM_RPC_URL $ARBISCAN_API_KEY $ARBITRUM_ADMIN"
-    chains["arbitrum_sepolia"]="$ARBITRUM_SEPOLIA_RPC_URL $ARBISCAN_API_KEY $ARBITRUM_SEPOLIA_ADMIN"
+    chains["arbitrum_sepolia"]="arbitrum_sepolia $ARBISCAN_API_KEY $TESTNET_ADMIN"
```

Comment:
it should be `$ARBITRUM_SEPOLIA_RPC_URL` and not `arbitrum_sepolia` in this script. We use [.env.deployment.example](https://github.com/sablier-labs/v2-core/blob/main/.env.deployment.example) to load custom RPCs as public ones may have rate limiting and may not work at the time of deployment.

Reply:
my idea with using `<chain>_sepolia` is to read the value from `foundry.toml`, it is similar to:

```bash
FOUNDRY_PROFILE=optimized \
forge script script/DeployDeterministicCore.s.sol \
--broadcast \
--verify \
--etherscan-api-key ARBISCAN_API_KEY \
--rpc-url arbitrum_sepolia \
--sig "run(address)" \
0xb1bEF51ebCA01EB12001a639bDBbFF6eEcA12B9F \
-vvv
```

Reply:
we don't need to re-declare it in `.env.deployment`

Reply:
Good point. Then, why don't we remove [.env.deployment](https://github.com/sablier-labs/v2-core/blob/main/.env.deployment) file entirely and only use `foundry.toml` for all the chains? Main network endpoints require `API_KEY_INFURA` in `foundry.toml` s so having `.env.deployment` is just redundant and more efforts to manage.

Reply:
i agree to use only the `foundry.toml` file, but instead of “forcing” someone to use `infura,` we should change it to `<CHAIN_NAME>_RPC_URL`. this way, you can use any RPC provider, such as alchemy

Reply:
In that case, they can also change it directly in `foundry.toml` file. My point was to have consistency in how we load those variables. `<CHAIN_NAME>_RPC_URL` works too but does that mean that these RPC URLs must be added to `.env` file? If yes, then you may want to update `.env.example` as well.

Reply:
> CHAIN_NAME>_RPC_URL works too but does that mean that these RPC URLs must be added  to .env file? If yes, then you may want to update .env.example as well:

yes, ofc

Reply:
Oh. I was suggesting to completely remove `.env.deployment.example` and only use `.env.example`. During fork test, there is no need to populate all rpc variables. So that deployment script loads from `foundry.toml` which loads from `.env`

Reply:
Agree to keep `.env.deployment.example`